README.md

```diff
@@ -1,15 +1,27 @@
-# Amaze File Manager
+# <a href="https://teamamaze.xyz">Amaze File Manager</a>
+
+<img height="50px" src="icon.png" data-canonical-src="icon.png" >
```

Comment:
I thing icon can be placed to the left of the title

Reply:
Will try, I think it had some weird vertical centerning issues though.

Reply:
![imagen](https://user-images.githubusercontent.com/10991116/114063123-a7902b80-986e-11eb-8435-b2eff0a22576.png)